# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/palo/larger-cd-dvd.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

Upstream palo erors out creating bootable ISOs larger than 800MB.
Increase that for 2GB for now. More might need more tweaks.

--- palo-2.24/palo/palo.c.vanilla	2023-12-12 22:16:24.689714475 +0100
+++ palo-2.24/palo/palo.c	2023-12-12 22:17:13.030710188 +0100
@@ -378,7 +378,7 @@
     struct firstblock f;
     /* always load IPL at second 2k block on sequential media */
     int where = 1 * FW_BLOCKSIZE;
-    unsigned end = 800 * 1024 * 1024;
+    unsigned end = 2000 * 1024 * 1024;
     /* int is safe since CD-ROM is < 2G */
     int bootloader_offset;
     int kernel32_offset;
